### PR TITLE
front: fix missing translation in manage vias modal title

### LINKED ISSUE #7494: front: missing translation on manage vias modal
### What happened?

Missing translation on manage vias modal
image

### OSRD version (top right corner `Account` button > `Informations`)

staging 3f7ff32

### PR DESCRIPTION
**Pull request by emersion**
ModalSuggestedVias has been updated with the correct title when the translation was switched to _zero/_other, but ModalSuggeredVias hasn't been updated. (Yes, these are two different names.)

Closes: https://github.com/OpenRailAssociation/osrd/issues/7494
Fixes: 83de94366b24 ("front: adapt itinerary for trainschedule v2")

### CODE REVIEW

File: front/src/modules/pathfinding/components/Itinerary/ModalSuggeredVias.tsx [hunk 1]
```diff
@@ -117,7 +117,7 @@ export default function ModalSugerredVias({
   return (
     <div className="manage-vias-modal">
       <ModalHeaderSNCF withCloseButton>
-        <h1>{`${t('manageVias')} ${vias.length > 0 ? `(${vias.length})` : ''}`}</h1>
+        <h1>{`${t('manageVias', { count: vias.length })}`}</h1>
```
Comment (clarani) [inline front/src/modules/pathfinding/components/Itinerary/ModalSuggeredVias.tsx]: You can even remove the extra string ;)
```suggestion
        <h1>{t('manageVias', { count: vias.length })}</h1>
```

**Comment by emersion** [comment]
I've noticed that the translation didn't work when there is exactly one via (only works for zero, or multiple). See the last commit's message:

    react-i18next's pluralization feature requires providing at least
    two variants: "_one" and "_other". An additional "_zero" variant
    may be provided, but doesn't change the fact that "_one" is
    required. This rule is broken by "manageVias".
    
    One way to fix this would be to just add the variant by copy-pasting
    "manageVias_other", however pluralization is designed to handle
    one vs multiple, not zero vs one-or-multiple. Indeed, the choice to
    hide/show the number of vias will never depend on the language. So
    it doesn't feel like the right tool for the job here.
    
    Instead, use a conditional in the react component.

<details>

<summary>Alternative fix</summary>

```diff
diff --git a/front/public/locales/en/operationalStudies/manageTrainSchedule.json b/front/public/locales/en/operationalStudies/manageTrainSchedule.json
index b0a672e1122d..16a4cb4e5032 100644
--- a/front/public/locales/en/operationalStudies/manageTrainSchedule.json
+++ b/front/public/locales/en/operationalStudies/manageTrainSchedule.json
@@ -57,6 +57,7 @@
   "launchPathFinding": "Launch pathfinding",
   "launchSimulation": "Start",
   "manageVias_zero": "Steps management",
+  "manageVias_one": "Steps management ({{count}})",
   "manageVias_other": "Steps management ({{count}})",
   "maximumRunTime": "Maximum running time",
   "maximumRunTimeError": "Travel time must not exceed 12h",
diff --git a/front/public/locales/fr/operationalStudies/manageTrainSchedule.json b/front/public/locales/fr/operationalStudies/manageTrainSchedule.json
index 231155b820e2..d663e5de1506 100644
--- a/front/public/locales/fr/operationalStudies/manageTrainSchedule.json
+++ b/front/public/locales/fr/operationalStudies/manageTrainSchedule.json
@@ -57,6 +57,7 @@
   "launchPathFinding": "Lancer la recherche d'itinéraire",
   "launchSimulation": "Démarrer",
   "manageVias_zero": "Gestion des étapes",
+  "manageVias_one": "Gestion des étapes ({{count}})",
   "manageVias_other": "Gestion des étapes ({{count}})",
   "maximumRunTime": "Temps de parcours maximum",
   "maximumRunTimeError": "Le temps de parcours ne doit pas excéder 12h",
```

</details>

File: front/src/modules/pathfinding/components/Itinerary/ModalSuggeredVias.tsx (hunk 1 above)
Comment (emersion) [reply front/src/modules/pathfinding/components/Itinerary/ModalSuggeredVias.tsx]: Good point! Also simplified TSv2.

**Review by SharglutDev — APPROVED** [review]
Lgtm. Agreed with this approach.

**Review by clarani — APPROVED** [review]

**Pull request merged**